eotsmanager/cmd/eotsd/daemon/keysadd.go

```diff
@@ -0,0 +1,396 @@
+package daemon
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/input"
+	"github.com/cosmos/cosmos-sdk/client/keys"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	cryptokeyring "github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/multisig"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/go-bip39"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+)
+
```

Comment:
maybe we can highlight somewhere that this part of code is adapted from Cosmos SDK and we add a perminent link here?

Reply:
done at https://github.com/babylonlabs-io/finality-provider/pull/148/commits/e03230b89f5ab74fc9eb189df0e9385e9ca41e1f